src/cascadia/TerminalSettingsEditor/Actions.cpp

```diff
@@ -19,24 +19,27 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     {
         InitializeComponent();
 
-        _filteredActions = winrt::single_threaded_observable_vector<winrt::Microsoft::Terminal::Settings::Model::Command>();
+        _filteredActions = winrt::single_threaded_observable_vector<Command>();

```

Comment:
this will become immediately obnoxious if you ever introduce `Editor::Command`

Reply:
I'm gonna not fix this if that's ok. I already have an Actions Page PR out. And I don't think I have that problem there because I introduced `KeyBindingViewModel` instead.